kubetest/kind/kind.go

```diff
@@ -237,3 +318,71 @@ func (k *Kind) KubectlCommand() (*exec.Cmd, error) {
 	cmd.Env = append(os.Environ(), fmt.Sprintf("KUBECONFIG=%s", kubeConfigPath))
 	return cmd, nil
 }
+
+func getFromURL(url string) ([]byte, error) {
+	timeout := time.Duration(60 * time.Second)
+	client := http.Client{
+		Timeout: timeout,
+	}
+	resp, err := client.Get(url)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+	return b, nil
+}
+
+// getKindRelease accepts either a kind release tag or 'kindBinaryLatest'.
+func getKindRelease(tag string) (*kindRelease, error) {
+	b, err := getFromURL("https://api.github.com/repos/kubernetes-sigs/kind/releases")
```

Comment:
this might hit a API request cap. i didn't find a tokenized client in kubetest.

Reply:
ok, will make changes here to make it possible to:
- build from latest kind master or specific sha/tag to produce a binary
or
- download a binary from a TAG hardcoded in the source code of the deployer
(kubetest/kubekins has to be bumped each time a new kind release is out).